code.user.js

```diff
@@ -488,6 +498,13 @@
                 error: function() {
                     return callback(ERROR_FAILED);
                 },
+                statusCode: {
```

Comment:
Seems weird to list all the status codes, I'd check for 429 in the error callback instead. (I think it goes into error callback?)

Reply:
made some changes, now statuscode checks via xhr.status